R/pkg/R/functions.R

```diff
@@ -259,6 +259,20 @@ setMethod("column",
           function(x) {
             col(x)
           })
+#' corr
+#'
+#' Computes the Pearson Correlation Coefficient for two Columns.
+#'
+#' @rdname corr
+#' @name corr
+#' @family math_funcs
+#' @export
+#' @examples \dontrun{corr(df$c, df$d)}
+setMethod("corr", signature(x = "Column", col1 = "Column", col2 = "missing", method = "missing"),
```

Comment:
as for doc, the DataFrame corr in stats.R has `@rdname statfunctions`
this one has `@rdname corr`

so they go to different HTML page generated by roxygen2